machine/tests/integration/config.py

```diff
@@ -11,7 +11,7 @@
 MINIO_APPLICATION_NAME = "minio"
 MINIO_S3_ACCESS_KEY = "vaultintegrationtest"
 MINIO_S3_SECRET_KEY = "vaultintegrationtest"
-NUM_VAULT_UNITS = 3
+NUM_VAULT_UNITS = 1
```

Comment:
Why are we making this change?

Reply:
Oops, this was just a change I made to test quicker -- and accidentally committed. Embarassing. :cry: